Source/WebCore/page/InteractionRegion.cpp

```diff
@@ -220,6 +220,7 @@ std::optional<InteractionRegion> interactionRegionForRenderedRegion(RenderObject
             bounds.expand(IntSize(borderBoxRect.size() - contentBoxRect.size()));
         }
     }
+    borderRadius = std::max(borderRadius, (float)regionRenderer.document().settings().interactionRegionMinimumCornerRadius());
```

Comment:
std::max<float>(...) instead of the C-style cast. Also maybe a newline above here.